File: Assignment___21/Assignment_3.c
#include<stdio.h>
#include<stdlib.h>

int Difference(int Arr[] , int iLength)
{
    int iCnt = 0 , iMin = Arr[0], iMax = Arr[0], iDiff = 0 ;
    for(iCnt = 0 ; iCnt < iLength;iCnt++)
    {
        if(iMin >= Arr[iCnt])
        {
            iMin = Arr[iCnt];
        }
        else if (iMax <= Arr[iCnt]) 
        {
            iMax = Arr[iCnt];
        }
    }
    printf("(%d - %d)\n",iMax, iMin);
    iDiff = iMax - iMin;
    return iDiff;

}
int main()
{
    int iSize = 0 , iCnt = 0 ;
    int iRet = 0 ;
    int *ptr = NULL;

    printf("Enter number of Elements :  ");
    scanf("%d",&iSize);

    ptr = (int *)malloc(iSize * sizeof(int));
    if(ptr == NULL)
    {
        printf("Unable to locate memory : \n");
        return -1;
    }

    printf("Enter %d numbers : \n",iSize);
    for(iCnt = 0  ; iCnt < iSize ; iCnt++)
    {
        printf("Enter %d numbers : ",iCnt+1);
        scanf("%d",&ptr[iCnt]);
    }

    iRet = Difference(ptr , iSize); 
    printf("Difference number is : %d \n",iRet);

    free(ptr);

    return 0 ;
}

File: Assignment__18/Assignment_java/Assignment_3.java

import java.util.*;

class Divible
{
    public void Display(int []Arr , int iLength)
    {
        for(int iCnt = 0 ; iCnt < iLength ; iCnt++)
        {
            if((Arr[iCnt] % 5 == 0 ) && (Arr[iCnt] % 2 == 0))
            {
                System.out.print(Arr[iCnt]+ " ");
            }
        }
        System.out.println();

    }
}
class Assignment_3
{

    public static void main(String A [])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter Number : ");
        int iSize = sobj.nextInt();
    

        int []ptr = new int[iSize];

        System.out.println("Enter "+iSize+" Elements : ");

        for(int iCnt = 0 ; iCnt < iSize; iCnt++)
        {
            System.out.print("Enter elements "+ (iCnt + 1) +" : ");
            ptr[iCnt] = sobj.nextInt();
        }
        Divible dobj = new Divible();
        dobj.Display(ptr , iSize);

    }
}